src/MudBlazor/Components/Input/MudInput.razor.cs

```
@@ -323,16 +323,15 @@ private bool HasNativeHtmlPlaceholder()
                 or InputType.Time or InputType.Week;
         }
 
-        /// <summary>
-        /// Releases resources used by this component.
-        /// </summary>
-        /// <returns></returns>
-        public async ValueTask DisposeAsync()
+        /// <inheritdoc />
+        protected override async ValueTask DisposeAsyncCore()
         {
             if (AutoGrow && IsJSRuntimeAvailable)
             {
                 await JsRuntime.InvokeVoidAsyncWithErrorHandling("mudInputAutoGrow.destroy", ElementReference);
```

Is there any chance this will throw an exception?    (Exceptions should not be raised during `DisposeAsync()`.)

---

The `InvokeVoidAsyncWithErrorHandling` was specifically written in a way not to throw exceptions.
It doesn't swallow all the exception, but all related to the JS:
- InvalidOperationException on pre-rendering exception when JS runtime doesn't exist.
- JSDisconnectedException when blazor server breaks the circuit.
- TaskCanceledException when cancellation token is cancelled.
- ObjectDisposedException when `DotNetObjectReference` was disposed.